.github/workflows/tests.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
           username: rudderlabs
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Warehouse Integration Test
-        run: make run-warehouse-integration
+        run: make run-warehouse-integration tags=warehouse_integration
```

Comment:
This is out of the scope of the PR, but we should consider removing tags for tests.